Makefile

```diff
@@ -0,0 +1,40 @@
+CC=g++
+AR=ar
+TARGET_DIR= ./build
+TARGET_LIB=libminifi.a
+TARGET_EXE=minifi
+CFLAGS=-O0 -fexceptions -fpermissive -Wno-write-strings -std=c++11 -fPIC -Wall -g -Wno-unused-private-field
+INCLUDES=-I./inc -I./src -I./test -I/usr/include/libxml2 -I/usr/local/opt/leveldb/include/
+LDDIRECTORY=-L/usr/local/opt/leveldb/out-static/ -L./build
```

Comment:
@benqiu2016 This out-static directory seems to be problematic on OS X.  Is additional setup needed beyond brew install leveldb.